azblob/highlevel.go

```diff
@@ -86,7 +86,7 @@ func UploadBufferToBlockBlob(ctx context.Context, b []byte,
 		if o.Progress != nil {
 			body = pipeline.NewRequestBodyProgress(body, o.Progress)
 		}
-		return blockBlobURL.Upload(ctx, body, o.BlobHTTPHeaders, o.Metadata, o.AccessConditions)
+		return blockBlobURL.Upload(ctx, body, o.BlobHTTPHeaders, o.Metadata, o.AccessConditions, ClientProvidedKeyOptions{})
```

Comment:
>ClientProvidedKeyOptions{} [](start = 91, length = 26)

is there a reason for not supporting CPK in the highlevel functions?

Reply:
Done